include/obmc_hypervisor.hpp

```diff
@@ -117,8 +117,8 @@ inline void connectHandler(const boost::system::error_code& ec)
 inline void requestRoutes(App& app)
 {
     BMCWEB_ROUTE(app, "/console1")
-        .privileges({{"ConfigureComponents", "ConfigureManager"}})
         .websocket()
+        .privileges({{"ConfigureComponents", "ConfigureManager"}})
```

Comment:
Change to OemIBMServiceAgent.  Only the service agent should be allowed to access the PHYP console.